src/collar/oc_particle.lsl

```diff
@@ -670,21 +705,7 @@ default {
 
     listen(integer iChannel, string sName, key kID, string sMessage) {
         if (iChannel == g_iChan_ocCmd){
-            list lOcCMD = llParseString2List(sMessage, [":"],[]);
```

Comment:
Oh I see the HandleOccCMD function just relocates this code.  Still would like to see a well defined generic protocol that any attachment could use but this refactor isn't really making that easier or harder than before.

Reply:
I made a plugin that was at first only for cuffs, but now it could be used for any attachment that needs chains/rlv. I plan to remove that plugin as soon as the v8 api is there, but the chain-code would then still be needed.